### ISSUE TITLE
Workflow: Allow for duplication of records incl. relationships

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#635]
This is meant as an enhancement Issue:

Currently it is not possible to create a workflow that duplicates an invoice incl. its line items. 
I can use workflow to duplicate the invoice fields themselves, but not the line items. Obviously an invoice without it's line-items is useless.

-> I'd really like to see this functionality added in the future!

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Workflow: Allow for duplication of records incl. relationships](https://trello.com/c/U1gcxFRO/137-workflow-allow-for-duplication-of-records-incl-relationships)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
